cms/templates/settings.html

```diff
@@ -199,6 +199,26 @@ <h3 class="title">${_("These Dates Are Not Used When Promoting Your Course")}</h
           </div>
           % endif
         </section>
+
+        <section class="group-settings details">
```

Comment:
In general we have hidden things that will be ignored by the Drupal site, as it is confusing for authors. They must supply that information in a different place, but by showing it on the Studio settings page, they THINK they are setting information that will actually be used-- in reality, it has no bearing on what Drupal shows.

I would recommend hiding it until the Drupal site is able to use the information. You could also check with product about this.